molo/core/middleware.py

```diff
@@ -116,10 +116,16 @@ def submit_tracking(self, account, request, response, custom_params={}):
         if hasattr(request, 'user') and hasattr(request.user, 'profile')\
                 and request.user.profile.uuid:
             uuid = request.user.profile.uuid
+
             params = build_ga_params(
                 request, account, path=path,
                 referer=referer, title=title,
                 user_id=uuid, custom_params=custom_params)
+        else:
+            params = build_ga_params(
+                request, account, path=path,
+                referer=referer, title=title,
+                custom_params=custom_params)
```

Comment:
@sewagodimo on line 111 it is already doing this. You can just add the `custom_params` to that call.

Reply:
@KaitCrawford I moved it up